worker/src/queues/cloudUsageMeteringQueue.ts

```diff
@@ -0,0 +1,61 @@
+import { Queue, Worker } from "bullmq";
+import logger from "../logger";
+import { redis, QueueName, QueueJobs } from "@langfuse/shared/src/server";
+import { instrumentAsync } from "../instrumentation";
+import { handleCloudUsageMeteringJob } from "../ee/cloudUsageMetering/handleCloudUsageMeteringJob";
+import { env } from "../env";
+
+export const cloudUsageMeteringQueue =
+  redis && env.STRIPE_SECRET_KEY
+    ? new Queue(QueueName.CloudUsageMeteringQueue, {
+        connection: redis,
+      })
+    : null;
+
+if (cloudUsageMeteringQueue) {
+  cloudUsageMeteringQueue.add(
+    QueueJobs.CloudUsageMeteringJob,
+    {},
+    {
+      repeat: { pattern: "5 * * * *" },
+    }
+  );
+
+  // add a job to the queue to start the job immediately in case we need to catch up
+  cloudUsageMeteringQueue.add(QueueJobs.CloudUsageMeteringJob, {}, {});
+
+  // log the jobs in the queue
+  cloudUsageMeteringQueue.getJobCounts().then((counts) => {
+    logger.info("Cloud Usage Metering Queue", counts);
+  });
+}
+
+export const cloudUsageMeteringJobExecutor =
+  redis && env.STRIPE_SECRET_KEY
+    ? new Worker(
+        QueueName.CloudUsageMeteringQueue,
+        async (job) => {
+          if (job.name === QueueJobs.CloudUsageMeteringJob) {
+            return instrumentAsync(
+              { name: "cloudUsageMeteringJobExecutor" },
```

Comment:
Shall i add this to dd dashboard or will you do this?

Reply:
it'd be great if you can do dashboard then